events/bingo/serverScripts/bingoWheel/bingoWheelServer.js

```diff
@@ -0,0 +1,248 @@
+                    gameOnLights.push(childEntity);
+                }
+            });
+        },
+        
+        /* GAME ON: Double check that game set lights are on, close registration, and put up the card remover sign */
+        lightsOn: function() {
+            gameOnLights.forEach(function(light) {
+                Entities.editEntity(light, { visible: true });
+            });
+            Entities.editEntity(_this.entityID, {textures: JSON.stringify({
+                "file2": "https://hifi-content.s3.amazonaws.com/jimi/environment/bingo/BingoBoardWheel2.fbx/Polychrome-D.png",
+                "file3": "https://hifi-content.s3.amazonaws.com/jimi/environment/bingo/BingoBoardWheel2.fbx/Polychrome-M.jpg",
+                "file4": "https://hifi-content.s3.amazonaws.com/jimi/environment/bingo/BingoBoardWheel2.fbx/Polychrome-R.jpg",
+                "file5": "https://hifi-content.s3.amazonaws.com/jimi/environment/bingo/BingoBoardWheel2.fbx/Polychrome-E.jpg"
+            })});
+            Entities.editEntity(backboard, {textures: JSON.stringify({
+                "file2": "https://hifi-content.s3.amazonaws.com/jimi/environment/bingo/BingoBoard2.fbx/Polychrome-D.png",
+                "file3": "https://hifi-content.s3.amazonaws.com/jimi/environment/bingo/BingoBoard2.fbx/Polychrome-M.jpg",
+                "file4": "https://hifi-content.s3.amazonaws.com/jimi/environment/bingo/BingoBoard2.fbx/Polychrome-R.jpg",
+                "file5": "https://hifi-content.s3.amazonaws.com/jimi/environment/bingo/BingoBoard2.fbx/Polychrome-E.jpg"
+            })});
+        },
+
+        /* OPEN REGISTRATION: Take down card remover sign and put up card spawner sign. */
+        openRegistration: function() {
+            Entities.editEntity(cardRemoverSign, {
+                script: "",
+                visible: false
+            });
+            Entities.editEntity(registrationSign, { visible: true });
+        },
+
+        /* CLOSE REGISTRATION: Take down card remover sign. */
+        closeRegistration: function() {
+            Entities.editEntity(registrationSign, { visible: false});
+        },
+
+        /* NEW ROUND: Clear the list of called numbers, turn on game set lights and turn off all bingo wall number lights. 
+        Clear any interval, close registration, and put up the card remover sign. */
+        newRound: function() {
+            newRoundURLParams = encodeURLParams({ 
+                type: "newRound",
+                calledNumbers: JSON.stringify(calledNumbers),
+                roundNumber: roundNumber
+            });
+            request({
+                uri: SPREADSHEET_URL + "?" + newRoundURLParams
+            }, function (error, response) {
+                if (error || !response) {
+                    print("SPREADSHEET URL: ", SPREADSHEET_URL + "?" + newRoundURLParams);
+                    return;
+                }
+                
+                Entities.editEntity(BINGO_WHEEL_TEXT, {
```

Comment:
similar comment as [here](https://github.com/highfidelity/hifi-content/pull/250/files#r248834039).

for this instance, make sure your response has a success code. we don't want this script to think the remote operation succeeded when it didn't.